_providers/dubby.org.md

```diff
@@ -0,0 +1,13 @@
+---
+name: dubby.org
+status: OK
+domains:
+- dubby.org
+strict_tls: true
+config_defaults:
+  sentbox_watch: 0
+  mvbox_watch: 0
+  mvbox_move: 0
```

Comment:
self-send is missing? i think the default is "off" but it should be on i think, to avoid at least one issue with multi-device usage.

Reply:
sorry, i'll add that.